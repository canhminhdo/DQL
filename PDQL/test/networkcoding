#!/bin/sh

DIR=../test
cd ../examples && maude < networkcoding.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/networkcoding.out
diff $DIR/networkcoding.expected $DIR/networkcoding.out
rm $DIR/networkcoding.out