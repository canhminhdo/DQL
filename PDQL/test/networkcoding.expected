==========================================
reduce in NETWORK-CODING : initQState |= [H(0) ; CX(0,1) ; H(2) ; CX(2,3) ; H(
    4) ; CX(4,5) ; H(6) ; CX(6,7) ; H(8) ; CX(8,9) ; H(10) ; CX(10,11) ; H(12)
    ; CX(12,13) ; network-coding ; SWAP(2,4) ; SWAP(0,4)]P(1,2,EPR) /\ P(4,5,
    EPR) .
result JudgmentSet: emptyJS
Bye.
